cluster/kube-up.sh

```diff
@@ -84,6 +84,7 @@ elif [[ "${validate_result}" == "2" ]]; then
 fi
 
 if [[ "${ENABLE_PROXY:-}" == "true" ]]; then
+  # shellcheck disable=SC1091
```

Comment:
shellcheck is used the right way
https://github.com/koalaman/shellcheck/wiki/SC1091